Source/WebKit/WebProcess/GPU/graphics/ImageBufferBackendHandle.h

```diff
@@ -38,7 +41,7 @@ using ImageBufferBackendHandle = std::variant<
     , MachSendRight
 #endif
 #if ENABLE(RE_DYNAMIC_CONTENT_SCALING)
-    , DynamicContentScalingDisplayList
+    , WebCore::DynamicContentScalingDisplayList
```

Comment:
I think I am confused here. Why is this considered a backend handle? I do not think there is an backend backed by a DisplayList.

Reply:
Sure there is; `DynamicContentScalingImageBufferBackend`, that's existing code (and it's what `DynamicContentScalingBifurcatedImageBuffer` uses to encapsulate the DL management. You could imagine moving that code into `DynamicContentScalingBifurcatedImageBuffer` but we can do that separately.